fetch/main.js

```diff
@@ -0,0 +1,16 @@
+async function grab() {
+  try {
+    const response = await fetch('https://pokeapi.co/api/v2/pokemon/1', {
```

Comment:
GET is the default, so for get requests you _could_ just leave off the entire 2nd argument.